src/Main.scala

```diff
@@ -0,0 +1,34 @@
+/**
+  * Created by halleyfroeb on 10/17/16.
+  */
+object Main {
+
+
+  def main(args: Array[String]): Unit = {
+
+    val rick =  Person("Rick", 65, "Male")
+    val morty =  Person("Morty", 11, "Male")
+    val summer =  Person("Summer", 17, "Female")
+    val jerry =  Person("Jerry", 42, "Male")
+
+    val peopleList = List(rick, morty, summer, jerry)
+    peopleList.foreach( (i: Person) => i.printPerson(i))
+
+    //same as above but different method
+
+    rick.printPerson(rick)
+    morty.printPerson(morty)
+    summer.printPerson(summer)
+    jerry.printPerson(jerry)
```

Comment:
FYI, doesn't this look simplier:   println(jerry)
You don't need to pass a parameter to a method in Person if the object is already a Person. This is the reason for overriding toString() like this:
..    override def toString: String = s"Name = $name, Age = $age, Gender = $gender\n"